cbmgmtx/mgmt.go

```diff
@@ -98,6 +108,39 @@ func (h Management) DecodeCommonError(resp *http.Response) error {
 	}
 }
 
+type InvalidArg struct {
+	field  string
+	reason string
+}
+
+func parseForInvalidArg(body string) InvalidArg {
```

Comment:
Is there some reason we can't just use json.Unmarshall for this?

Reply:
None other than the smoothness of my brain. Changed now